rpcs3/Emu/RSX/Overlays/overlay_perf_metrics.cpp

```diff
@@ -11,26 +11,57 @@ namespace rsx
 {
 	namespace overlays
 	{
+		void perf_metrics_overlay::reset_transform(label& elm) const
+		{
+			constexpr u32 margin_pos = 50;
+			position pos;
+
+			switch (m_quadrant)
+			{
+			case screen_quadrant::top_left:
+				pos.x = margin_pos;
+				pos.y = margin_pos;
+				break;
+			case screen_quadrant::top_right:
+				pos.x = virtual_width - margin_pos - m_body.w;
+				pos.y = margin_pos;
+				break;
+			case screen_quadrant::bottom_left:
+				pos.x = margin_pos;
+				pos.y = virtual_height - margin_pos - m_body.h;
+				break;
+			case screen_quadrant::bottom_right:
+				pos.x = virtual_width - margin_pos - m_body.w;
+				pos.y = virtual_height - margin_pos - m_body.h;
+				break;
+			default: ;
+			}
+
+			elm.set_pos(pos.x, pos.y);
+			elm.set_margin(5, 5, 0, 0);
+			elm.set_padding(0, 0, 0, -14);
+		}
+
+		void perf_metrics_overlay::reset_transforms()
+		{
+			reset_transform(m_body);
```

Comment:
wont this set both m_body and m_titles to the same position? or is this intended behaviour?

Reply:
Intended behaviour, they need to be right on top of each other (the only reason for titles existing as a separate `label` is to style them differently).